app/decorators/application_decorator.rb

```diff
@@ -0,0 +1,36 @@
+class ApplicationDecorator < Draper::Decorator
+  def status_tag
+    tag = h.govuk_tag(text: status_text.html_safe, colour: status_colour)
+    tag += unpublished_status_hint if object.has_unpublished_changes?
+    tag.html_safe
+  end
```

Comment:
Is it worth adding a TODO here suggesting we refactor this into a component at some point?

Reply:
We need a giant TODO to refactor everything